mungegithub/mungers/sig-mention-handler_test.go

```diff
@@ -0,0 +1,162 @@
+				State: nil,
+				Labels: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+					githubapi.Label{Name: githubapi.String(sigApps)}},
+				PullRequestLinks: nil,
+				Assignee:         &githubapi.User{Login: githubapi.String(username)},
+				Number:           intPtr(1),
+			},
+			expected: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+				githubapi.Label{Name: githubapi.String(sigApps)}},
+		},
+		{
+			name: "issue is closed",
+			issue: &githubapi.Issue{
+				State: githubapi.String("closed"),
+				Labels: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+					githubapi.Label{Name: githubapi.String(sigApps)}},
+				PullRequestLinks: nil,
+				Assignee:         &githubapi.User{Login: githubapi.String(username)},
+				Number:           intPtr(1),
+			},
+			expected: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+				githubapi.Label{Name: githubapi.String(sigApps)}},
+		},
+		{
+			name: "issue has at least one sig label",
+			issue: &githubapi.Issue{
+				State: githubapi.String(open),
+				Labels: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+					githubapi.Label{Name: githubapi.String(sigApps)}},
+				PullRequestLinks: nil,
+				Assignee:         &githubapi.User{Login: githubapi.String(username)},
+				Number:           intPtr(1),
+			},
+			expected: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+				githubapi.Label{Name: githubapi.String(sigApps)}},
+		},
+		{
+			name: "issue has no sig label",
+			issue: &githubapi.Issue{
+				State:            githubapi.String(open),
+				Labels:           []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)}},
+				PullRequestLinks: nil,
+				Assignee:         &githubapi.User{Login: githubapi.String(username)},
+				Number:           intPtr(1),
+			},
+			expected: []githubapi.Label{githubapi.Label{Name: githubapi.String(helpWanted)},
+				githubapi.Label{Name: githubapi.String(needsSigLabel)}},
+		},
+	}
+
+	for testNum, test := range tests {
+		client, server, mux := github_test.InitServer(t, test.issue, nil, nil, nil, nil, nil, nil)
+		config := &github.Config{}
+		config.Org = "o"
+		config.Project = "r"
+		config.SetClient(client)
+		mux.HandleFunc(fmt.Sprintf("/repos/o/r/issues/1/labels/%s", needsSigLabel), func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+		})
+
+		obj := &github.MungeObject{}
+		var err error
+
+		if test.issue != nil {
+			obj, err = config.GetObject(*test.issue.Number)
+
+			if err != nil {
+				t.Fatalf("%d: unable to get issue: %v", testNum, *test.issue.Number)
+			}
+
+			obj.Issue = test.issue
+		}
+
+		s := SigMentionHandler{}
+		s.Munge(obj)
+
+		if obj.Issue == nil {
+			if test.expected != nil {
+				t.Errorf("%d:%s: expected: %v, saw: %v", testNum, test.name, test.expected, nil)
+			} else {
+				continue
+			}
+		}
+
+		if len(test.expected) != len(obj.Issue.Labels) {
```

Comment:
Yes, `DeepEqual` will cover length difference. i am following format of a test file written before. i think checking the length difference first can be more efficient if there are many items involved. i will remove this check. :)